packages/docs/src/pages/pricing.tsx

```diff
@@ -337,12 +337,12 @@ export default function PricingPage(): JSX.Element {
               </li>
               <li id="note7">
                 Data usage refers to the creation of{' '}
-                <a href="https://www.medplum.com/docs/tutorials/api-basics/create-fhir-data">FHIR Resources</a>. This
-                figure is cumulative.
+                <a href="https://www.medplum.com/docs/fhir-datastore/create-fhir-data">FHIR Resources</a>. This figure
```

Comment:
I think this should be a relative link, no?

Reply:
Relative doesn't work in the JSX elements